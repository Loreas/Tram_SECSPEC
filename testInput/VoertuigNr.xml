<SYSTEM>
    <station>
        <naam>A</naam>
        <spoor>
            <spoorNr>12</spoorNr>
            <volgende>C</volgende>
            <vorige>C</vorige>
        </spoor>
        <type>halte</type>
    </station>
    <tram>
        <lijnNr>12</lijnNr>
        <zitplaatsen>50</zitplaatsen>
        <snelheid>50</snelheid>
        <beginStation>A</beginStation>
        <type>PCC</type>
        <voertuigNr></voertuigNr>
    </tram>
</SYSTEM>
