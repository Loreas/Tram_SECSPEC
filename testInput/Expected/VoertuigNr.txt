======================
Starting XML Parser...
======================

Creating new Station:
---------------------
	Getting Station name...
		Station Name: A
	Getting Station Track...
		Track Number: 12
	Getting Next Station...
		Next Station: A
	Getting Previous Station...
		Previous Station: A
	Getting Station Type...
		Station Type: halte

Creating new Tram:
------------------
	Getting Tram Number...
		Tram Line Number: 12
	Getting Tram seats...
		Tram Seats: 50
	Getting Tram speed...
		Tram Speed: 50
	Getting Starting Station...
		Starting Station: A
	Getting Current Station...
		Current Station: A
	Getting Tram Type...
		Tram Type: PCC
	Getting Tram Vehicle Number...
		FAULT IN TRAM VEHICLE NUMBER
DELETING INCOMPLETE TRAM

===================
XML Parser finished
===================

